pkgs/development/python-modules/dicom2nifti/default.nix

```diff
@@ -13,21 +13,29 @@
 
 buildPythonPackage rec {
   pname = "dicom2nifti";
-  version = "2.3.0";
-  disabled = isPy27;
+  version = "2.4.3";
+  disabled = pythonOlder "3.6";
 
   # no tests in PyPI dist
   src = fetchFromGitHub {
     owner = "icometrix";
     repo = pname;
     rev = version;
-    sha256 = "sha256-QSu9CGXFjDpI25Cy6QSbrwiQ2bwsVezCUxSovRLs6AI=";
+    hash = "sha256-za2+HdnUhPu3+p29JsF4iL1lyPQVmEv3fam0Yf1oeMQ=";
   };
 
-  propagatedBuildInputs = [ nibabel numpy pydicom scipy setuptools ];
+  propagatedBuildInputs = [ gdcm nibabel numpy pydicom scipy setuptools ];
 
-  checkInputs = [ nose gdcm ];
-  checkPhase = "nosetests tests";
+  # python-gdcm just builds the python interface provided by the "gdcm" package, so
+  # we should be able to replace "python-gdcm" with "gdcm" but this doesn't work; maybe #84774?
```

Comment:
```suggestion
  # we should be able to replace "python-gdcm" with "gdcm" but this doesn't work; maybe https://github.com/NixOS/nixpkgs/issues/84774 ?
```

Reply:
yes, metadata is missing. Not sure how this is supposed to work.
```
$ ls result/lib/python3.10/site-packages/
 _gdcmswig.so   gdcm.py   gdcmswig.py
```

Reply:
It seems like GDCM is broken in a way similar to VTK (and likely some other Python packages built as part of a larger (CMake?) build?).  (Version 2.3.0 appears to depend on `gdcm` through CLI tools so wasn't affected by the metadata issue.)

I've inlined the issue URL.